en/rocky/8/guides/RL_rsnapshot_backup.md

```diff
@@ -0,0 +1,361 @@
+# Rocky Linux - Backup Solution - Rsnapshot
+
+# Introduction
+
+Rsnapshot is a very powerful backup utility that can be installed on any Linux-based machine. It can either backup a machine locally, or you can backup multiple machines, say servers for instance, from a single machine. Rsnapshot uses rsync and is written entirely in perl with no library dependencies, so there are no weird requirements to installing it. In the case of Rocky Linux, you should be able to install rsnapshot simply by installing the EPEL repository. This documentation covers the installation of rsnapshot on Rocky Linux only.
+
+## Prerequisites
```

Comment:
(Maybe a style guide issue) Maybe have a sentence like "To successfully understand and implement the instructions in this documentation, we assume you already:"